tests/components/application-connector/k8s/gateway-test/templates/test-app.yml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-app
```

Comment:
Let's rename it to something like `mock-service` or `mock-application`

Reply:
Will change it to mock-application